SpeedTest.cpp

```diff
@@ -344,10 +344,14 @@ std::map<std::string, std::string> SpeedTest::parseJSON(const std::string &data)
 
     obj = json::JSON::Load(data);
     try {
-        map["ip_address"] = (std::ostringstream() << obj["ip"]).str();
-        map["isp"] = (std::ostringstream() << obj["company"]["name"]).str();
-        map["lat"] = (std::ostringstream() << obj["location"]["latitude"]).str();
-        map["lon"] = (std::ostringstream() << obj["location"]["longitude"]).str();
+        map["ip_address"] = static_cast<std::ostringstream &&>
```

Comment:
Try this instead to correctly dump the strings without extra quotes:

```cpp
    try {
        map["ip_address"] = obj["ip"].ToString();
        map["isp"] = obj["company"]["name"].ToString();
        map["lat"] = obj["location"]["latitude"].dump();
        map["lon"] = obj["location"]["longitude"].dump();
    } catch(...) {}
```

Reply:
I have never tried to output to JSON, so your patch corrects it as well as brings to the code more readability. 
Thanks for the fix!